src/server/api/routers/tdarr.ts

```diff
@@ -0,0 +1,375 @@
+      TranscodeDecisionMaker: z.string(),
+      file: z.string(),
+      file_size: z.number(),
+      container: z.string(),
+      video_codec_name: z.string(),
+      video_resolution: z.string(),
+    })
+  ),
+  totalCount: z.number(),
+});
+
+export type TdarrQueue = {
+  array: {
+    id: string;
+    healthCheck: string;
+    transcode: string;
+    file: string;
+    fileSize: number;
+    container: string;
+    codec: string;
+    resolution: string;
+    type: 'transcode' | 'health check';
+  }[];
+  totalCount: number;
+  startIndex: number;
+  endIndex: number;
+};
+
+export const tdarrRouter = createTRPCRouter({
+  statistics: publicProcedure
+    .input(inputSchema)
+    .query(async ({ input }): Promise<TdarrStatistics> => {
+      const app = getTdarrApp(input.appId, input.configName);
+      const appUrl = new URL('api/v2/cruddb', app.url);
+
+      const body = {
+        data: {
+          collection: 'StatisticsJSONDB',
+          mode: 'getById',
+          docID: 'statistics',
+        },
+      };
+
+      const res = await axios.post(appUrl.toString(), body);
+      const data = getStatisticsSchema.parse(res.data);
+
+      return {
+        totalFileCount: data.totalFileCount,
+        totalTranscodeCount: data.totalTranscodeCount,
+        totalHealthCheckCount: data.totalHealthCheckCount,
+        failedTranscodeCount: data.table3Count,
+        failedHealthCheckCount: data.table6Count,
+        stagedTranscodeCount: data.table1Count,
+        stagedHealthCheckCount: data.table4Count,
+        pies: data.pies.map((pie) => ({
+          libraryName: pie[0],
+          libraryId: pie[1],
+          totalFiles: pie[2],
+          totalTranscodes: pie[3],
+          savedSpace: pie[4] * 1_000_000_000, // file_size is in GB, convert to bytes,
+          totalHealthChecks: pie[5],
+          transcodeStatus: pie[6],
+          healthCheckStatus: pie[7],
+          videoCodecs: pie[8],
+          videoContainers: pie[9],
+          videoResolutions: pie[10],
+          audioCodecs: pie[11],
+          audioContainers: pie[12],
+        })),
+      };
+    }),
+
+  workers: publicProcedure.input(inputSchema).query(async ({ input }): Promise<TdarrWorker[]> => {
+    const app = getTdarrApp(input.appId, input.configName);
+    const appUrl = new URL('api/v2/get-nodes', app.url);
+
+    const res = await axios.get(appUrl.toString());
+    const data = getNodesResponseSchema.parse(res.data);
+
+    const nodes = Object.values(data);
+    const workers = nodes.flatMap((node) => {
+      return Object.values(node.workers);
+    });
+
+    return workers.map((worker) => ({
+      id: worker._id,
+      file: worker.file,
+      fps: worker.fps,
+      percentage: worker.percentage,
+      ETA: worker.ETA,
+      jobType: worker.job.type,
+      status: worker.status,
+      step: worker.lastPluginDetails?.number ?? '',
+      originalSize: worker.originalfileSizeInGbytes * 1_000_000, // file_size is in MB, convert to bytes,
```

Comment:
The value is called "originalfileSizeInGbytes", shouldn't the multiplier be for Gbytes then?
Is it giving a value in Mbytes despite having a mention for Gbytes in the name?